src/unity/python/turicreate/toolkits/drawing_classifier/drawing_classifier.py

```diff
@@ -138,6 +138,15 @@ def create(input_dataset, target, feature=None, validation_set='auto',
     _raise_error_if_not_drawing_classifier_input_sframe(
         input_dataset, feature, target)
 
+    if batch_size is not None and not isinstance(batch_size, int):
```

Comment:
I believe several other methods take a batch_size parameters. Do you want to make these same checks there too?